### PR TITLE
Front issues resolution

### PR DESCRIPTION
**Pull request by sachabinder**
- Photo/club photo front profile
- Order of buttons to change profile in personal profile page
- Post front : Display date of each post, Title of the post, placeholder on comment,  date under comment, usable button on comment author, add event button, like and comment zone.
- fix bug on comment delete button

### CODE REVIEW

File: back/news/templates/news/event_detail.html [hunk 1]
```diff
@@ -89,12 +89,12 @@ <h4 class="page-title">Informations sur l'évènement : {{event.name}}</h4>
                 <!--Column-->
                 <div class="col-detail-lg-6">
                     <h4 class="page-title">Posts liés à cet évènement</h4>
-
                     {% if is_member %}
-                        <div class="centered-div mt-2">
-                            <h5 class="text-bold mb-2">Ajouter un post</h5>
-                            <a href="{% url 'news:post_create_with_origin' event.id%}"><i class="fas fa-plus-circle fa-5x"></i></a>
-                        </div>
+                    <div class="centered-div mt-2">
```
Comment (StephaneGrabsia) [inline back/news/templates/news/event_detail.html:93]: Don't you think we should indent this block for clarity purpose?

**Review by StephaneGrabsia — APPROVED** [review]
CR ok, good job

**Comment by LudoBike** [comment]
# Post create/edit
It looks quite awful on my computer (cf screenshot). Space is missing under selection lists and the selection lists are wider than the other inputs. I also think it is strange that the description input is centered rather than left aligned.

image

# Posts
I remark that H1 in description are bigger than the title of the post but it is not necessarily an issue. 

However the name of the linked event is not written and I find that strange. Perhaps we can specify "Ce post est lié à l'évènement : [nom]" before the post title or between the title and the description. Otherwise it is still better than the old version.

image

# Profile
It's great, perhaps we can just add a test to display the nickname only if it is different from the full name, otherwise the same info is written twice. 

image

And I find strange that the navbar highlight "Profil" rather than "Élèves" when we look at the profil of another user.

image

# Profile edit
Similarly to post edit, the selection list are wider than the other inputs.

image

**Review by LudoBike — COMMENTED** [review]
cf https://github.com/KIClubinfo/upont-v4/pull/120#issuecomment-1257183964

**Review by TheoContentin — COMMENTED** [review]
Agreeing with Stéphane, change are all ok to me, typo in CSS can be problematic

**Comment by sachabinder** [comment]
Some other things ?

**Review by LudoBike — CHANGES_REQUESTED** [review]
Add name of related event in post card

**Review by LudoBike — APPROVED** [review]

**Pull request merged**